src/components/Home/NoticeFunSystemTab/LineSearchBar/LineSearchBar.style.ts

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled';
+import Image from 'next/image';
+import { TEXT_STYLES } from '@/styles/constants/textStyles';
+import {COLORS} from '@/styles/constants/colors';
+
+export const SearchBarBox = styled.div`
+  width : 100%;
+  height :56px;
+  top : 108px;
+  left : 0px;
+  padding-left : 16px;
+  padding-right : 16px;
+  padding-bottom : 8px; 
```

Comment:
padding 값을 지정해주는건 어떤가요?
```suggestion
  padding: 0 16 8 16;
```